CardanoSharp.Blazor/CardanoSharp.Blazor.Components/WalletConnector.razor

```diff
@@ -209,10 +220,76 @@ else
         //set up initial _wallets state from above
         _selfReference = DotNetObjectReference.Create(this);
         _wallets = await _walletConnectorJs.Init(SupportedExtensions, _selfReference);
+        
         Initialized = true;
+
+        await WalletOnInitialized();
+
         return;
     }
 
+    private async Task WalletOnInitialized()
+    {
+        var supportedWalletKeys = SupportedExtensions?.Select(s => s.Key)?.ToArray();
+
+        if (supportedWalletKeys != null && supportedWalletKeys.Length > 0)
+        {
+            var storedWalletKey = await HasLocalStorageWalletKey(supportedWalletKeys);
+
+            if (!string.IsNullOrWhiteSpace(storedWalletKey))
+            {
+                if (!await ConnectWalletAsync(storedWalletKey, false))
+                {
+                    await RemoveStoredWalletKey();
+                }
+            }
+        }
+
+        Connecting = false;
+
+        StateHasChanged();
+    }
+
+    private async Task<string> HasLocalStorageWalletKey(params string[] supportedWalletKeys)
```

Comment:
please add Async as a method name suffix for async methods

Reply:
also maybe rename this to HasStoredWalletKey so its following the same convention as the others